src/badguy/mrbomb.cpp

```diff
@@ -1,5 +1,7 @@
-//  SuperTux
+//  SuperTux BadGuy MrBomb
```

Comment:
I noticed we don't modify this part of the copyright header anymore. Should we modify it here?